### ISSUE TITLE
[Feature Request] Reverse the direction of the progress bar when reading right to left

### ISSUE THREAD

**Issue by Darkwolf1515** [gotson/komga#299]
### Is your feature request related to a problem? Please describe.
Bar go right when book go left

### Describe the solution you'd like
Make bar go left when book go left
### Describe alternatives you've considered or other apps that can do what you want
Cover, google play books

**Comment by gotson** [comment]
Blocked by https://github.com/vuetifyjs/vuetify/issues/12132

**Comment by KucharczykL** [comment]
Would be nice to have this since upstream made it possible.

**Comment by gotson** [comment]
> Would be nice to have this since upstream made it possible.

Not exactly, it's only in vuetify 3, Komga uses vuetify 2. Vuetify 3 is not yet fully ready.
